frontend/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@shopgate/pdp-native-share",
+  "version": "0.1.0",
+  "description": "",
+  "license": "Apache-2.0",
+  "scripts": {
+    "lint": "eslint --parser babel-eslint --ignore-path ../.gitignore --ignore-path .eslintignore --ext .js --ext .jsx .",
+    "test": "jest",
+    "test:watch": "npm run test -- --watch",
+    "coverage": "jest --coverage",
+    "coverage:watch": "npm run coverage -- --watch",
+    "dummy": ""
+  },
+  "devDependencies": {
+    "@shopgate/eslint-config": "*",
```

Comment:
Also new thing, we need to change pwa dependencies to `peerDependencies` where you can define what is the minimum required version for the extension.

I guess it would be 5.9.0-beta-something or 5.10

Reply:
Added peer dependencies for pwa-common, commerce, core, ui-ios, ui-material